app/views/shared/_navbar.html.erb

```diff
@@ -15,14 +15,18 @@
         <% end %>
```

Comment:
Make sure to update with main, I think a few things on the navbar have been updated